recipes/platform.interfaces/all/conanfile.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class PlatformInterfacesConan(ConanFile):
+    name = "platform.interfaces"

```

Comment:
```suggestion
    name = "linksplatform-interfaces"
```

User author+project name for generic names or not very popular projects

Reply:
I mean, it's a naming convention for packages, we use author's name, not namespace because is safer in terms of avoiding name collision. We already had problems with generic project names, we don't want to repeat the same situation.

Still, the namespace for cmake, pkgconfig, ... is not a problem, using [components](https://docs.conan.io/en/latest/creating_packages/package_information.html#using-components) you can use any custom name, including Platform.Interfaces.

Reply:
Here I would say that the name of the library is `platform.interfaces`, I don't see any problem with it... it is not because of disambiguation, it is just the name of the lib.

Reply:
So what do we need to do to use Platform as a prefix for our packages? We have to register a trademark? We have to become popular? How popular?

Reply:
> So what do we need to use Platform as a prefix for our packages?

We can just take it first
https://conan.io/center/search/platform

Reply:
Searching on Github, Gitlab and Sourceforge, I didn't find any related name, only tags or projects relates to hardware, so I'm fine using `platform.interfaces`. Let's revert to the original name.

Still, the project name is `interfaces`, but the namespace is `platform.interfaces`. I would suggest using `platlform.interfaces` as project name too. Github redirects automatically after renaming a project.

Reply:
C++ CMake project is called `Platform.Interfaces` https://github.com/linksplatform/Interfaces/blob/181e070877c176374d2ad1da50bebf1614ceb6da/cpp/CMakeLists.txt#L5
which does not match the repository yet others have the same naming convention

Reply:
So we cannot leave the repository name `Interfaces`, the namespace name `Platform.Interfaces` and the package name `platform.interfaces` at the same time?

Reply:
You can use what name you want, but consider renaming to `Platform.Interfaces` as you are using as main name. Not only because package names, but also because of search engines result, like Github. If search for Platform.Interfaces, that project won't be listed at first page.

Reply:
@uilianries we can use uppercase letters in the package name?

Reply:
No, only lowercase is allowed. The Bot checks it when exporting the package (1st step on building) and breaks if it's not lowercase.

Why? Case sensitive problems when installing packages, [here](https://github.com/conan-io/conan-center-index/issues/3302) is a real example of many others.

Reply:
@Konard we are just sharing our thoughts as active members/consumers/contributors of open source software :) 

It was not an issue for our CCI rules hence why I approved.

Hope to see more of your projects added to CCI! 🚀